### DISCUSSION TITLE
Make the URI to be recognized as the same after the query string

### DISCUSSION THREAD

**Discussion by mtaishiro** [manugarg/pagenotes-issues#70] [Ideas]
Currently, if the query string changes, page notes recognize the website as a different one, which makes it difficult to track the notes. For example, [http://www.example.com?foo=bar&foo1=bar1](http://www.example.com/?foo=bar&foo1=bar1) and [http://www.example.com](http://www.example.com) is currently recognized to be a different website. Instead of saving the full URI, if you could fix it to save the one without query string, it would help a lot.

**Comment by DavidSchor** [comment]
If that change is added I would like it to be a setting. For me and my workflows those 2 would be different pages and therefor have different context for notes.  So the suggested fix would be destructive to that.

**Reply by manugarg** [reply]
Ack. Of course, @DavidSchor!

**Comment by manugarg** [comment]
Note that this idea is same as the feature request filed here:
https://github.com/manugarg/pagenotes-issues/issues/41

I support this idea, but can you please share examples of websites where it will be beneficial? I know it will make a lot of sense for Google Docs for example. I am looking for other examples.

**Comment by manugarg** [comment]
I just wanted to share that I am working on this. It should be out in a couple of weeks.

Follow up in #41.

**Comment by manugarg** [comment]
Folks this has been done. It's available from 3.4.4 onwards. See https://pagenotes.manugarg.com/faq/ignore_in_url/ for this works.
